gtars/src/uniwig/cli.rs

```diff
@@ -70,4 +70,12 @@ pub fn create_uniwig_cli() -> Command {
                 .help("Number of rayon threads to use for parallel processing")
                 .required(false),
         )
+        .arg(
+            Arg::new("score")
+                .long("score")
+                .short('o')
+                .value_parser(clap::value_parser!(bool))
+                .help("Count via score (narrowPeak only!)")
+                .required(false),
+        )
```

Comment:
I mentioned already but can we simply make the presence of this flag represent `true`? So that one doesn't need to say `-o true`?

Reply:
Done.